scripts/build.sh

```diff
@@ -2,8 +2,8 @@
 
 # This script builds the application from source for multiple platforms.
 # Determine the arch/os combos we're building for
-ALL_XC_ARCH="386 amd64 arm arm64 ppc64le"
-ALL_XC_OS="linux darwin windows freebsd openbsd solaris"
```

Comment:
Adding those ARCH LGTM, but I'm not sure we can remove solaris support that easily.
What do you think @SwampDragons ? I think we should merge this PR like this then I can do a PR to add a `make bin` CI step and reintroduce solaris.

Reply:
As long as we re-add support before we do the next release, I'm okay with that.

Reply:
Got it, merging !